CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Features
+
+- Added sentry cli path as parameter for actions [Finalize PR #42] ([#97](getsentry/sentry-fastlane-plugin/pull/97))
```

Comment:
```suggestion
- Added sentry cli path as parameter for actions [Finalize PR #42](#97)
```

Reply:
Seems that Danger does not like the current format. This is what's suggested.

Reply:
Ah, I see, makes sense.
https://github.com/getsentry/sentry-fastlane-plugin/blob/master/dangerfile.js#L2

Let me add a space before `(`!